src/main/java/com/epam/tc/gdi/hw1/site/pages/components/simple/HiddenCheckList.java

```diff
@@ -0,0 +1,27 @@
+package com.epam.tc.gdi.hw1.site.pages.components.simple;
+
+import com.epam.jdi.light.common.JDIAction;
+import com.epam.jdi.light.elements.complex.Checklist;
+import java.util.Arrays;
+import java.util.List;
+import java.util.stream.Collectors;
+import org.openqa.selenium.By;
+import org.openqa.selenium.WebElement;
+
+public class HiddenCheckList extends Checklist {
```

Comment:
Checklist can be used if use locator to input

Reply:
Yes, [type="checkbox"] is input elements.